tests/tools/collab/test_e2e_divergence.py

```
@@ -0,0 +1,536 @@
+
+
+def test_a_deletion_during_a_pull_is_not_undone(collab, faults):
+    """Test that a tombstone recorded while a delta is in flight is honoured by the import that lands it.
+
+    The pull holds a state object across a handshake, a negotiation and a download — minutes of wall clock — and
+    the import is what has to see the tombstones as they are when it runs, not as they were before any of it.
+    Both defences fall to the same staleness: the claim was computed before the deletion, so the node is cut into
+    the delta, and the import's tombstone filter would read the same stale set. Pull-only: only the pull carries
+    a state object across a network round trip; the endpoint already re-reads its own inside the import lock.
+    """
+    a, b, c = collab(3)
```

Ruff (0.16.1)

[warning] Unpacked variable `c` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)